index.js

```diff
@@ -3778,6 +3778,31 @@
   S.pairs =
   def('pairs', {}, [$.StrMap(a), $.Array($.Pair($.String, a))], pairs);
 
+  //# fromPairs :: Foldable f => f (Pair String a) -> StrMap a
+  //.
+  //. Returns a string map containing the key–value pairs specified by
+  //. the given [Foldable][]. If a key appears in multiple pairs, the rightmost
```

Comment:
Nit: Let's make the lengths of the two lines above more even by moving the first word of the second line to the end of the first line. :)